iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/ShortBackupRetentionCheck.java

```diff
@@ -156,8 +168,8 @@ private static String toString(@Nullable Tree tree) {
     return Optional.ofNullable(tree)
```

Comment:
It's not actually part of this PR, but for me, a more clear name for this method would be something like `propertyValueAsString` or `getValueAsString`. `toString` feels a bit too generic.
Also, there is some overlapping with methods from `PropertyUtils`, but I couldn't think of a way to use them here, since we don't have a common supertype for `StringLiteral` and `StringComplete`